### ISSUE TITLE
[Accessibility] [Global] [Charts] [Area (simple)] > Plot points are not keyboard operable or announced by screen readers

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2439]
[A11Y-613](https://jira.cengage.com/browse/A11Y-613)

Environment:
Windows 11 / Chrome 137 / JAWS 2025
Windows 11 / Edge 136 / NVDA 2025
MacOS 15 / Safari / VoiceOver

Steps to Reproduce:
1. Open the URL - https://react-magma.cengage.com/version/4.9.1/data-visualization/chart-demos/.
2. Turn on a screen reader (JAWS/NVDA/VoiceOver).
3. Attempt to navigate to individual data points (plots) using the keyboard (Tab/Arrow keys).
4. Observe whether the tooltip content is accessible and announced.

Actual Result:
Plot points are not focusable via keyboard.
Tooltip content (e.g., data values) is only available on mouse hover and not accessible to screen reader users.
No semantic role or label is provided for the data points.

Refer to Screencast: https://app.screencast.com/hIoLjsbwWNL7Q

Instance: This applies to all the sub components

Expected Result:
Each plot point should be keyboard focusable (e.g., using tabindex="0").
When focused, the screen reader should announce the pointwith tooltip content.
For example:-
"Plot 1 button 2019 Annual Sales Jan 1 2019,netc.. "

User Impact:
Keyboard and screen reader users cannot access the data conveyed by the chart, excluding them from critical information and insights.

WCAG Guidelines:
2.1.1 Keyboard (Level A)

Bug Fix Recommendation:
Make each data point focusable using tabindex="0",
Add role="button" and add aria-label to each point with a meaningful description of the data.
Ensure tooltip content is accessible via keyboard and screen reader (use aria-describedby attribute)

**Comment by olehnoskov-cengage** [comment]
Closed as dublicated https://github.com/cengage/react-magma/issues/2338
